pyproject.toml

```diff
@@ -38,7 +42,17 @@ license = "MIT"
 authors = ["Peter Schutt <peter.github@proton.me>"]
 readme = "README.md"
 repository = "https://github.com/topsport-com-au/starlite-saqlalchemy"
-keywords = ["api", "rest", "http", "asgi", "starlite", "saq", "sqlalchemy", "plugin", "python"]
+keywords = [
```

Comment:
Is a linter doing this? I'd rather add a linter to our tools in a separate PR if that's something you think we need.